app/backend/src/couchers/servicers/groups.py

```diff
@@ -0,0 +1,180 @@
+import logging
+
+import grpc
+from sqlalchemy.sql import literal
+
+from couchers import errors
+from couchers.db import session_scope
+from couchers.models import Cluster, Node, Page, PageType, User
+from couchers.servicers.pages import _page_to_pb
```

Comment:
Remove underscore or do not import internal functions.

Reply:
Fixed for now by removing the underscore/internalness. This is something that needs to be evaluated in the long run. Also awaiting Lucas's comments on whether this should just return page ids instead of full pages.